third_party/terraform/tests/resource_cloud_identity_group_test.go.erb

```diff
@@ -0,0 +1,73 @@
+package google
+
+<% autogen_exception -%>
+<% unless version == 'ga' -%>
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccCloudIdentityGroup_update(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"org_domain":    getTestOrgDomainFromEnv(t),
+		"cust_id":       getTestCustIdFromEnv(t),
+		"random_suffix": randString(t, 10),
+	}
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProvidersOiCS,
+		CheckDestroy: testAccCheckCloudIdentityGroupDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCloudIdentityGroup_basic(context),
+			},
+			{
+				Config: testAccCloudIdentityGroup_update(context),
+			},
+		},
+	})
+}
+
+func testAccCloudIdentityGroup_basic(context map[string]interface{}) string {
```

Comment:
This is the same as `testAccCloudIdentityGroup_cloudIdentityGroupsBasicExample`, right? want to just use that?

Reply:
Ah, yes! I thought the random suffixes would be different between the two, but just tried it and it worked. Thanks!